python/cugraph/cugraph/tests/structure/test_graph.py

```diff
@@ -204,6 +205,31 @@ def test_add_adj_list_to_edge_list(graph_file):
     compare_series(destinations_cu, destinations_exp)
 
 
+@pytest.mark.sg
+def test_create_undirected_graph_from_asymmetric_adj_list():
+    # karate_asymmetric.get_path()
+    Mnx = utils.read_csv_for_nx(karate_asymmetric.get_path())
+    N = max(max(Mnx["0"]), max(Mnx["1"])) + 1
+    Mcsr = scipy.sparse.csr_matrix((Mnx.weight, (Mnx["0"], Mnx["1"])), shape=(N, N))
+
+    offsets = cudf.Series(Mcsr.indptr)
+    indices = cudf.Series(Mcsr.indices)
+
+    G = cugraph.Graph(directed=False)
+
+    with pytest.raises(Exception):
+        # Ifan undirected graph is created with 'symmetrize' set to False, the
+        # edgelist provided by the user must be symmetric.
+        G.from_cudf_adjlist(offsets, indices, None, symmetrize=False)
+
+    G = cugraph.Graph(directed=False)
+    G.from_cudf_adjlist(offsets, indices, None, symmetrize=True)
+
```

Comment:
Should this test at least assert that the number of edges is correct?

Reply:
This is the issue I highlighted in the FIXME; We currently do not have a mechanism at the API layer to extract the symmetric edges